pkg/services/sqldb/types.go

```diff
@@ -13,25 +13,27 @@ type ServerProvisioningParams struct {
 type DBProvisioningParams struct {
 }
 
-// TODO: Fix this...
-// Instance details are common to the DB only and All In One Service.
-// Bind doesn't get passed reference instance info, so we need the
-// Server Name, and Administrator Info in order to complete Binding
-type mssqlInstanceDetails struct {
+type mssqlAllInOneInstanceDetails struct {
 	ARMDeploymentName          string `json:"armDeployment"`
+	FullyQualifiedDomainName   string `json:"fullyQualifiedDomainName"`
```

Comment:
I re-ordered the structs so they . had everything declared in the same order to make commonality between them clear.